fcl/reco/Stage0/Run2/stage0_run2_icarus_mc_shiftForTrigger_refactored.fcl

```diff
@@ -0,0 +1,9 @@
+#include "stage0_run2_icarus_mc.fcl"
+
+
+physics.producers.mcophit.SimPhotonsProducer: "shiftIonization"
+physics.producers.ophit.InputModule: "shiftIonization"
+physics.producers.opdetonbeam.Waveforms: "shiftIonization"
+physics.producers.pmtbaselines.OpticalWaveforms: "shiftIonization"
+physics.producers.pmtfixedthr.OpticalWaveforms: "shiftIonization"
+physics.producers.triggersimgates.BeamGateTag: "shiftIonization"
```

Comment:
This seems to be the same as `stage0_run2_icarus_mc_shiftForTrigger.fcl` -- is it missing a `_refactored` in the include in line 1 maybe?

Reply:
I decided, here, to copy and modify [stage0_run2_icarus_mc_refactored.fcl](https://github.com/SBNSoftware/icaruscode/blob/525f06b16ff203e4bf13f4b265f855d3d5d333ba/fcl/reco/Stage0/Run2/stage0_run2_icarus_mc_refactored.fcl), and have a parallel version rather than continuing the chain of includes. If it's preferred, I can modify this fhicl but the difference would simply be to add `_refactored` to the include on line 1. The original fhicl is only two lines: including the non-refactored stage0 fhicl and pointing `mcophit.SimPhotonsProducer` to `pdfastsim`.

Reply:
Is it cleaner if we actually just make
```
stage0_run2_icarus_mc_shiftForTrigger_refactored.fcl
```
literally just be a one-liner?
```
#include stage0_run2_icarus_mc_shiftForTrigger.fcl
```

Maybe invoking @PetrilloAtWork for his insight on the intentions of fhicl maintainers?

Reply:
Now that I stare at these two files side-by-side they are actually identical, and we may not need to have separate fhicls here.

Reply:
The only reason they were separate previously, was because mcophit was (incorrectly) pointed to `pdfastsim`, but now that I fixed this they are completely redundant.

Reply:
Yeah that's why I was initially asking if something was missing 😅 - But the parallelism argument is fair.

Reply:
I tested that either file works for whichever version of the simulation chain is used, which makes sense as they are identical. Therefore, I've chosen to remove the redundant fhicl for simplicity. See: https://github.com/SBNSoftware/icaruscode/pull/673/commits/b8919bc2a9d80c33fa1bd72910c226743582fd11

Reply:
> Maybe invoking @PetrilloAtWork for his insight on the intentions of fhicl maintainers?

I do not have such an insight, unfortunately. My expectation is that `refactored` is a temporary label, and that after a (short?) while everything with `run3` will be implicitly refactored, but I can't guess about Run2 — I think it depends a lot on what the new MC production will use.